examples/extensions/commands/run_example.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+from test.examples_tools import run, tmp_dir
+
+
+non_deterministic_conanfile = """\
+from datetime import datetime
+
+from conan import ConanFile
+from conan.tools.files import copy
+
+
+class HelloConan(ConanFile):
+    name = "{name}"
+    version = "1.0"
+    {comment}
+
+    def build(self):
+        with open("random.txt", "w") as f:
+            f.write(str(datetime.now()))
+
+    def package(self):
+        # Packaging a random content, the package revision will be different every time
+        copy(self, "random.txt", self.source_folder, self.package_folder)
+"""
+
+
+def install_clean_command():
+    clean_command = os.path.join(os.path.dirname(os.path.realpath(__file__)), "clean")
```

Comment:
I don't think we should scope this. I would leave explaining the custom commands scope for cases where you want to group multiple commands in one category, for example if you have multiple commands for ci, like:
`ci:promote`, `ci:upload_built_packages`, ...

Reply:
Right, the commented scope above was just about the example location, to leave space for more examples, not about scoping the command itself, right?